UI/src/app/services/backend-connector.service.ts

```diff
@@ -11,10 +12,10 @@ export class BackendConnectorService {
     private readonly connectionTimeout = 100000;
 
 
-    private _data: BehaviorSubject<Map<number, Trace[]>> = new BehaviorSubject<Map<number, Trace[]>>(new Map());
+    private _data: BehaviorSubject<Map<number, Trace>> = new BehaviorSubject<Map<number, Trace>>(new Map());
```

Comment:
missing [ ]
I know it is fixed on the other branch, but I ll note it down so I don't forget to check again